css/style.css

```
@@ -0,0 +1,353 @@
+  text-align: center;
+}
+
+.user-navigation {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 21px;
+  line-height: 21px;
+  background-color: var(--basic-white);
+  text-align: right;
+}
+
+.user-navigation a {
+  color: var(--basic-black);
+}
+
+/*main*/
+
+
+.catalog-item h2{
+  font-family: Cuprum;
+font-style: normal;
+font-weight: bold;
+font-size: 24px;
+line-height: 30px;
+color: var(--basic-white)
+
+}
+.material {
+  background-color: var(--special-ylellow);
+}
+
+.tools {
+  background-color: var(--basic-blue);
+}
+
+.tech {
+  background-color: var(--basic-purple);
+}
+
+.sales {
+  background-color: var(--green);
+}
+
+.deliver {
+  background-color: var(--ylellow);
+}
+.catalog-slide{
+  background-image: url("../img/true-man.jpg");
+   background-repeat: no-repeat;
+}
+
+.popular-staff {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 30px;
+  line-height: 30px;
+  color: var(--basic-black);
+  background-color: var(--special-gray);
+}
+
+.drill-title {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--basic-black)
+}
+
+.old-price {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 18px;
+  color: var(--old-price);
+}
+
+.buy-button {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 18px;
+  color: var(--basic-white);
+  background-color: var(--basic-red);
+}
+
+.firms-list,
+.drill-list {
+  list-style: none;
+}
+
+.popular-firms-section,
+.popular-product-section {
+  background-color: var(--basic-white);
+}
+
+.about-item {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 13px;
+  line-height: 24px;
+  color: var(--basic-black);
+}
+
+.button-toogle {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 21px;
+  line-height: 30px;
+  color: var(--basic-white);
+  background-color: var(--address);
+}
+
+.deliv {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 30px;
+  line-height: 30px;
+  color: var(--address);
+  text-transform: uppercase;
+}
+
+.section-about {
+  background-color: var(--basic-white);
+}
+
+/*Footer*/
+.section-footer-first {
+  background-color: var(--address);
+}
+
+.section-footer-second {
+  background-color: var(--focus-nav);
+}
+
+.color-nav-footer a {
+  color: var(--basic-background);
+}
+
+.color-nav-footer-second a {
+  color: var(--gold);
+}
+
+.address-footer {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  color: var(--basic-background);
+}
+
+.socials {
+  list-style: none;
+  text-align: center;
+}
+
+.copiright,
+.feedback, {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 18px;
+}
+.copiright,
+.lead{
```

```suggestion
.lead {
```
Формат. Пройдись по всему файлу. Такое ещё встречается в некоторых местах 🙂